eZ/Publish/SPI/Persistence/Content/Section/Handler.php

```diff
@@ -94,4 +94,22 @@ public function assign($sectionId, $contentId);
      * @return int
      */
     public function assignmentsCount($sectionId);
+
+    /**
+     * Number of role policies using a Section in limitations.
+     *
+     * @param mixed $sectionId
+     *
+     * @return int
+     */
+    public function policiesCount($sectionId);
+
+    /**
+     * Counts the number of role assignments using section with $sectionId in their limitations.
+     *
+     * @param int $sectionId
+     *
+     * @return int
+     */
+    public function roleAssignmentsUsingSectionCount($sectionId);
```

Comment:
The original convention is countContentObjectsUsingSection, can maybe reflect that form on the new once also.